src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/management/TileTestsIT.java

```diff
@@ -0,0 +1,214 @@
+    String tileId = getTileId(organizationId, 0);
+    String imageId = getTileImageId(organizationId, tileId, 0);
+    
+    given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{ORGANIZATIONID}/tiles/{TILEID}/images/{IMAGEID}", organizationId, tileId, imageId)
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id", is(imageId))
+      .body("contentType", is(IMAGE_JPEG));
+  }
+  
+  @Test
+  public void testTileImageNotFound() {
+    String organizationId = getOrganizationId(0);
+    String tileId = getTileId(organizationId, 0);
+    String incorrectTileId = getTileId(organizationId, 2);
+    String imageId = getTileImageId(organizationId, tileId, 0);
+    
+    String[] malformedIds = new String[] {"evil", "*", "/", "1", "-1", "~"};
+    assertFound(String.format("/organizations/%s/tiles/%s/images/%s", organizationId, tileId, imageId));
+    assertEquals(1, countApiList(String.format("/organizations/%s/tiles/%s/images", organizationId, tileId)));
+    
+    for (String malformedId : malformedIds) {
+      assertNotFound(String.format("/organizations/%s/tiles/%s/images/%s", organizationId, tileId, malformedId));
+    }
+
+    assertNotFound(String.format("/organizations/%s/tiles/%s/images/%s", organizationId, incorrectTileId, imageId));
+    assertEquals(0, countApiList(String.format("/organizations/%s/tiles/%s/images", organizationId, incorrectTileId)));
+  }
+
+  @Test
+  public void testTileImageData() {
+    String organizationId = getOrganizationId(0);
+    String tileId = getTileId(organizationId, 0);
+    String imageId = getTileImageId(organizationId, tileId, 0);
+
+    given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{ORGANIZATIONID}/tiles/{TILEID}/images/{IMAGEID}/data", organizationId, tileId, imageId)
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .header("Content-Length", "21621")
+      .header("Content-Type", IMAGE_JPEG);
+  }
+  
+  @Test
+  public void testTileImageDataScaled() {
+    String organizationId = getOrganizationId(0);
+    String tileId = getTileId(organizationId, 0);
+    String imageId = getTileImageId(organizationId, tileId, 0);
+
+    given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{ORGANIZATIONID}/tiles/{TILEID}/images/{IMAGEID}/data?size=100", organizationId, tileId, imageId)
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .header("Content-Length", "13701")
+      .header("Content-Type", IMAGE_PNG);
+  }
+
+  private void createPtvSettings() {
+    insertSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL, String.format("%s%s", getWireMockBasePath(), BASE_URL));
+    flushCache();
+  }
+  
+  private void deletePtvSettings() {
+    deleteSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL);
+  }
+    
+  private void createManagementSettings(String organizationId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "createManagementSettings" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)